_topic_maps/_topic_map_rosa.yml

```diff
@@ -1606,6 +1601,124 @@ Topics:
   - Name: Glossary
     File: logging-common-terms
 ---
+Name: Backup and restore
+Dir: backup_and_restore
+Distros: openshift-rosa
+Topics:
+- Name: Overview of backup and restore operations
+  File: index
+- Name: Shutting down a cluster gracefully
+  File: graceful-cluster-shutdown
+# File: graceful-shutdown
+- Name: Restarting a cluster gracefully
+  File: graceful-cluster-restart
+# File: graceful-restart
+- Name: Installing OADP on ROSA with STS
+  File: install-oadp-rosa-sts
```

Comment:
Why we need to move it here? I see we already have https://docs.openshift.com/rosa/cloud_experts_tutorials/cloud-experts-deploy-api-data-protection.html
and it mentions:
```
This content is authored by Red Hat experts, but has not yet been tested on every supported configuration.
```
And this is not officially supported, we have https://issues.redhat.com/browse/XCMSTRAT-785 tracking the works.